src/app/shared/directives/digit-only.directive.ts

```diff
@@ -0,0 +1,17 @@
+import {Directive, HostListener} from '@angular/core';
+
+@Directive({
+  selector: '[appDigitOnly]'
```

Comment:
rename to vtmDigitOnly

Reply:
tslint wants me to put app instead of vtm